File: src/java/cliParams.java
public class cliParams {
    public static void main(String[] args) {
        if (args.length == 0) {
            System.out.println("No parameters");
        } else if (args.length == 1) {
            System.out.println("Parameter is " + args[0]);
        } else {
            System.out.println("Too many parameters");
        }
    }
}

File: src/cs/cliParams.cs
using static System.Console;

class Program
{
	public static void Main(string[] args)
	{
        if (args.Length == 0)
        {
            WriteLine("No paramters");
        } else if (args.Length == 1)
        {
            WriteLine("Parameter is " + args[0]);
        } else
        {
            WriteLine("Too many parameters");
        }
	}
}